src/Core/Content/Seo/SeoUrlPlaceholderHandler.php

```diff
@@ -72,7 +72,9 @@ private function createDefaultMapping(array $matches): array
     {
         $mapping = [];
         foreach ($matches as $match) {
-            $mapping[$match] = str_replace(self::DOMAIN_PLACEHOLDER, '', rtrim($match, '#'));
+            // remove self::DOMAIN_PLACEHOLDER from start
+            // remove # from end
+            $mapping[$match] = substr($match, 32, -1);
```

Comment:
I guess you have to still do rtrim or check that it ends with # or? 🤔 

![Uploading image.png…]()

Reply:
The preg_match_all only matches everything EXCEPT `#` until `#` occurs, so we can savely assume that `#` is the last character if I didn't miss something? 🤔
https://regex101.com/r/cTZwzw/1